packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -5596,7 +5596,7 @@ Expected response body:
 
 - Endpoint: `head /type/model/visibility`
 
-Generate abd send put model with write/create properties.
+Generate abd send put model with query properties.
```

Comment:
based off of the url, it seems more that we are testing `head` calls, not the query properties

Reply:
This is an operation left over from two years ago. The visibility folder mainly tests the impact of @visibility on properties in the model and should not include tests for head, I will remove this operation to ensure clear functional separation.

Reply:
I'm assuming the old test still serves it's purpose bc we are testing the affect of `head` on visibility. Additionally, removing it will be a breaking change which will have to be communicated to all emitter authors. I would recommend just adding your test as a separate test